Doc set for VMFR (#110937)
openshift/openshift-docs: modules/oadp-vmfr-accessing-files-web.adoc

// Module included in the following assemblies:
//
// backup_and_restore/application_backup_and_restore/oadp-vmfr/oadp-using-vmfr.adoc

:_mod-docs-content-type: PROCEDURE
[id="oadp-vmfr-accessing-files-web_{context}"]
= Accessing restored files through a web browser

[role="_abstract"]

> Access restored virtual machine files through a web browser by using the file browser interface provided by the `VirtualMachineFileRestore` (VMFR) CR. This helps you to browse, preview, and download files from VM backups.

Reviewer: ```suggestion
Access restored virtual machine (VM) files through a web browser by using the file browser interface provided by the `VirtualMachineFileRestore` (VMFR) custom resource (CR). You can browse, preview, and download files from VM backups.
```

just trying to be more user centric

.Prerequisites

* You are logged in to the cluster with the `cluster-admin` role.
* You have created a `VirtualMachineFileRestore` (VMFR) CR with the `fileAccess.fileBrowser` section configured.

> * The VMFR CR `status.phase` is `Completed`.

Reviewer: ```suggestion
* You are logged in to the cluster with the `cluster-admin` role.
* A `VirtualMachineFileRestore` (VMFR) CR with the `fileAccess.fileBrowser` section configured exists.
* The VMFR CR `status.phase` is `Completed`.
```

state-checks in prerequisites: "you have created..." might improve readability and consistency.
